Mathlib/Algebra/Group/Action/Defs.lean

```diff
@@ -354,6 +354,16 @@ lemma smul_smul_smul_comm [SMul α β] [SMul α γ] [SMul β δ] [SMul α δ] [S
     [IsScalarTower α β δ] [IsScalarTower α γ δ] [SMulCommClass β γ δ] (a : α) (b : β) (c : γ)
     (d : δ) : (a • b) • c • d = (a • c) • b • d := by rw [smul_assoc, smul_assoc, smul_comm b]
 
+@[to_additive]
+lemma smul_mul_smul_comm [Mul α] [Mul β] [SMul α β] [IsScalarTower α β β]
```

Comment:
How is this different from the existing `smul_mul_smul`?

Reply:
It is more general

Reply:
(deprecating that lemma, if they're truly the same, sounds find to me)